app/models/conjecture.rb

```diff
@@ -2,17 +2,16 @@
 
 # The Conjecture model
 class Conjecture < Sentence
-  EVALUATION_STATES = %w(not_yet_enqueued enqueued processing
-                         finished_successfully finished_unsuccessfully).freeze
-  REASONING_STATUSES ||= %w(OPN ERR UNK RSO THM CSA CSAS CONTR).freeze
+  PROOF_STATUSES ||= %w(OPN ERR UNK RSO THM CSA CSAS CONTR).freeze
```

Comment:
Why is this conditionally assigned?

Reply:
I'm not sure any more. I think this file is required many times and from the second require statement on, the system complains about a redefined constant.